pkg/quarterdeck/db/errors.go

```diff
@@ -0,0 +1,10 @@
+package db
+
+import "errors"
+
+var (
+	ErrTokenMissingEmail = errors.New("token is missing email address")
+	ErrTokenExpired      = errors.New("token has expired")
+	ErrInvalidSecret     = errors.New("invalid secret for token verification")
+	ErrTokenInvalid      = errors.New("token has invalid signature")
```

Comment:
```suggestion
	ErrTokenMissingEmail = errors.New("email verification token is missing email address")
	ErrTokenExpired      = errors.New("email verification token has expired")
	ErrInvalidSecret     = errors.New("invalid secret for email token verification")
	ErrTokenInvalid      = errors.New("email verification token has invalid signature")
```

We have a lot of tokens in Quarterdeck; may help debugging if we specify.